tests/FakeItEasy.Specs/FakeOptionsBuilderSpecs.cs

```diff
@@ -118,6 +118,7 @@ public class FakeOptionsBuilderSpecs
                 .x(() => exception.Should().BeAnExceptionOfType<ExpectationException>());
         }
 
+        [Scenario]

```

Comment:
I know! I accidentally killed it a few PRs ago. I think during the big language cleanup. I can find the reference, if you want.